src/Http/Environment.php

```diff
@@ -233,11 +194,8 @@ public function detect(array $options = null, array $info = null): array
 	/**
 	 * Sets the host name, port, path and protocol from the
 	 * fixed list of allowed URLs
-	 *
-	 * @param array|string $allowed
-	 * @return void
 	 */
-	protected function detectAllowed($allowed): void
+	protected function detectAllowed(array|string|object $allowed): void
```

Comment:
Why `object`? Won't this be wrapped into `[object]` and then tried to be passed to `new Uri()` (which only accepts `array|string`)?